raster/r.watershed/front/r.watershed.md

```diff
@@ -151,9 +151,9 @@ from article entitled *Revised Slope Steepness Factor for the Universal
 Soil Loss Equation* (McCool et al. 1987). Since the S factor is a small
 number (usually less than one), the GRASS output map is of type DCELL.
 
-### A<sup>T</sup> least-cost search algorithm
+### A^T least-cost search algorithm
 
-*r.watershed* uses an A<sup>T</sup> least-cost search algorithm (see
+*r.watershed* uses an A^T least-cost search algorithm (see
```

Comment:
GitHub's rich rendering was better with the html syntax.

GitHub's markdown flavor has even better math markup support, like latex. We could explore what our tooling supports later on. 

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions

It doesn't _have_ to be changed now, I still agree with what is done in the PR.

Reply:
By default mkdocs [doesn't seem to support it](https://www.markdownguide.org/tools/mkdocs/), probably there is some extension.

Reply:
Ok, fine!